app/src/test/java/com/codesoom/assignment/controllers/UserAuthenticateControllerTest.java

```diff
@@ -0,0 +1,134 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.AuthenticationService;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserModificationData;
+import com.codesoom.assignment.infra.JpaUserRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class UserAuthenticateControllerTest {
+    private String invalidToken;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private JpaUserRepository jpaUserRepository;
+
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private String validToken;
+
+    @DisplayName("update 메서드")
+    @Nested
+    class Describe_update {
+        private User savedUser;
+        private String requestBody;
+
+        @BeforeEach
+        void setUp() throws JsonProcessingException {
+            savedUser = jpaUserRepository.save(User.builder()
+                    .email("a@a.com")
+                    .name("김코드")
+                    .password(passwordEncoder.encode("12345"))
+                    .build()
+            );
+
+            UserModificationData userModificationData = UserModificationData.builder()
+                    .name("김 숨")
+                    .password("098765")
+                    .build();
+
+            requestBody = objectMapper.writeValueAsString(userModificationData);
+
+            validToken = "Bearer " + authenticationService.login(savedUser.getEmail(), "12345");
```

Comment:
별 거 아니라고 생각할 수 있지만 `Bearer `는 토큰이 아닙니다. 이렇게 해보면 어떨까요?
```suggestion
            authorizationHeader = "Bearer " + authenticationService.login(savedUser.getEmail(), "12345");
```

Reply:
https://github.com/CodeSoom/spring-week7-assignment-1/pull/76/commits/aef1e2e47a086972a1b9a93ede6a95d642fcf613

수정했습니다